lib/runtime-core/src/instance.rs

```diff
@@ -449,7 +457,7 @@ impl InstanceInner {
                 let imported_func = &self.import_backing.vm_functions[imported_func_index];
                 (
                     imported_func.func as *const _,
-                    Context::External(imported_func.vmctx),
+                    Context::External(unsafe { imported_func.func_ctx.as_ref() }.vmctx.as_ptr()),
```

Comment:
Why does this need `unsafe`?

Reply:
[`NonNull::as_ref`](https://doc.rust-lang.org/std/ptr/struct.NonNull.html#method.as_ref) is marked unsafe.